backend/app/services/open_calls/destroy.rb

```diff
@@ -0,0 +1,25 @@
+module OpenCalls
+  class Destroy
+    attr_accessor :open_call, :emails
+
+    def initialize(open_call)
+      @open_call = open_call
+      @emails = []
+    end
+
+    def call
+      collect_emails
+      open_call.destroy!
+      emails.map(&:deliver_later)
+    end
+
+    private
+
+    def collect_emails
+      emails << InvestorMailer.open_call_destroyed(open_call.investor, open_call.name)
+      open_call.project_developers.each do |project_developer|
```

Comment:
Is it possible that project developer applies twice with different project for the same open call? I just tested this and got 2 same emails for one project developer, the project name that applied for open call is not in the emails, so the emails were the same. I wonder if we should do a distinct here or change email text, or it should not be possible to apply twice. Dunno :)

Reply:
Hmm, there is restriction where you cannot apply for same open call with same project, but you can apply for same open call with different project. To by honest, I dont know what is right solution here :). Do you have some idea @agnessa or should I ask at Slack? From my point of view, the easiest is to add project name to email...

Reply:
I agree, easiest to inject project name somewhere in the email 👍